client/src/pages/Message/MessageWindow/OwnerBubble/OwnerBubble.tsx

```diff
@@ -0,0 +1,17 @@
+import { Box, Typography } from '@material-ui/core';
+import useStyles from './useStyles';
+
+export interface OwnerTextProps {
+  text: string;
+}
+
+export default function OwnerBubble({ text }: OwnerTextProps): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Box className={classes.root}>
```

Comment:
Same as comment above for MUI Box component

Reply:
added CSS property inside the Box component.